package battle.exceptions

class BattleNotSetUpException : Exception("Call method setup before forwarding the battle")
